updater/bin/fetch_files.rb

```diff
@@ -32,6 +32,10 @@ class UpdaterKilledError < StandardError; end
 end
 
 begin
+  if Dependabot::Environment.job_id.to_i.even?
```

Comment:
would it help to have a feature flag here? That'll make rollback easier (toggling a flag versus a code change)

Reply:
We rollback by specifying the sha via chatops, so the rollback ergonomics are very similar 🙂 YJIT is native to Ruby (and a default for Rails 7.2 apps), so I don't anticipate needing to rollback.

Reply:
I assume we don't have any flighting support for core? Not asking you to change anything, mostly trying to build context.

Reply:
I don't know what flighting support is. Can you explain?

Reply:
Oops; I meant feature flags (used that interchangeably)

Reply:
We do support feature flags, or something very similar. They're called Experiments in this codebase. They're registered/wired up via the service.